src/components/application/application.component.jsx

```diff
@@ -0,0 +1,27 @@
+import "./application.style.scss";
+import InactiveScriptModal from "./modal/modal.component";
+
+const Application = ({ data }) => {
```

Comment:
I am not very confident on this type of usage, lets take and action item to use strict type checking, it is not a hard requirement right now but still necessary for future.